internal/inbox/inbox.go

```
@@ -430,11 +508,39 @@ func (m *Manager) Update(id int, inbox imodels.Inbox) (imodels.Inbox, error) {
 		return imodels.Inbox{}, envelope.NewError(envelope.GeneralError, m.i18n.T("globals.messages.somethingWentWrong"), nil)
 	}
 
-	// Update the inbox in the DB.
+	tx, err := m.db.Beginx()
+	if err != nil {
+		return imodels.Inbox{}, m.persistenceError("starting inbox update", err)
+	}
+	defer tx.Rollback()
```

golangci-lint (2.13.2)

[error] Error return value of `tx.Rollback` is not checked

(errcheck)